Library/Formula/dcd.rb

```diff
@@ -0,0 +1,13 @@
+require "formula"
+
+class Dcd < Formula
+  homepage "https://github.com/Hackerpilot/DCD"
+  url "https://github.com/Hackerpilot/DCD.git", :using => :git, :tag => "v0.5.1"
```

Comment:
Are there submodules involved? If there aren't, we should be using the ` archive ` tarball - If there are submodules involved and this must be based on a ` :tag ` please add the ` :revision ` (commit) the tag was based on. The ` :using => :git ` is redundant here as well - Homebrew understands Github git calls.